### PR TITLE
Drillthrough and dashboards page documentation

### PR DESCRIPTION
**Pull request by mazameli**
This primarily adds a new section to the user's guide to explain the new drill through, filter, break out, and question-level actions. I put it towards the beginning of the user's guide, because I think it represents an easier way to start using Metabase than creating a new question from scratch.

This PR also updates the dashboard doc to reflect the changes we've made to dashboard organization and archiving.

Resolves https://github.com/metabase/metabase/issues/5001 and https://github.com/metabase/metabase/issues/5000

### CODE REVIEW

**Comment by mazameli** [comment]
I think I need to shrink down a few of the images in this PR. Taking screenshots on the laptop's retina screen does weird things sometimes.

File: docs/users-guide/03-basic-exploration.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Additionally, the question detail page has an Explore button in the bottom-right of the screen with options that change depending on the kind of question you're looking at. (Note that the Explore button disappears if your cursor stops moving.)
+
+![Action menu](images/drill-through/actions.png)
+
+Here's a list of all the actions:
+* Table actions
+  - `Count of rows by time` lets you see how many rows there were in this table over time.
+  - `Summarize this segment` gives you options of various summarization functions (sum, average, maximum, etc.) you can use on this table to arrive at a number.
+* Chart and pivot table actions
+  - `Break outs` will be listed depending on the question, and include the option to break out by a category, location, or time. For example, if you're looking at the count of total orders over time, you might be able to further break that out by customer gender, if that information is present.
```
Comment (salsakran) [inline docs/users-guide/03-basic-exploration.md]: Did we ever settle on a replacement word for "breakout" or "pivot"? Or are we going to kick that can down the road?

**Review by salsakran — APPROVED** [review]

File: docs/users-guide/03-basic-exploration.md (hunk 1 above)
Comment (mazameli) [inline docs/users-guide/03-basic-exploration.md]: To me, "break out" is still the most colloquial of the terms we've thrown out there. (E.g., "revenue broken out by state" sounds conversational to me.)

Comment (salsakran) [inline docs/users-guide/03-basic-exploration.md]: sounds good

Comment (salsakran) [inline docs/users-guide/03-basic-exploration.md]: or at least punting sounds good =)

**Review by kdoh — APPROVED** [review]

**Pull request merged**